test/ui/time-users-id-tasks.js

```diff
@@ -0,0 +1,27 @@
+const puppeteer = require("puppeteer");
+var captureTime = 40;
+(async () => {
+    const uiLocation = "https://nscstrdevusw2thucommon.z5.web.core.windows.net/#/users/6/tasks ";
+    const browser = await puppeteer.launch();   
+    var startDate = new Date();
+    var totalSeconds = 0;     
+    var startTime = startDate.getTime();
+
+    for(var i=0; i<captureTime; i++){
+        const page = await browser.newPage();
+        console.log(`Page opened, going to ${uiLocation}`);
+
+        await page.goto(uiLocation)
+
+        console.log(`Navigated to ${uiLocation}, taking screenshot`);
+            await page.screenshot();
```

Comment:
Save the screenshots to the repo so we can be sure this is working! I definitely had a lot of screenshots of "Loading..." or the spinning circle, so I had to change my `page.got(url)` to `page.goto(url, waitUntil="networkidle2")` to fix it.

Reply:
Toddy told me we don't need to store the screenshots that's why